src/transformStateWithClones.js

```diff
@@ -65,7 +65,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const transformedStates = [];
+  const newState = { ...state };
+
+  for (let i = 0; i < transforms.length; i++) {
+    switch (transforms[i].operation) {
+      case 'addProperties' :
+        Object.assign(newState, transforms[i].properties);
+        transformedStates.push(Object.assign({}, newState));
```

Comment:
you can also use spread operator like in line 69

Reply:
use spread, line 69